### PR TITLE
Add Create button on Playa

### PR DESCRIPTION
**Pull request by dmitrymarchuk**
Closes https://github.com/sparkletown/internal-sparkle-issues/issues/1127

### CODE REVIEW

File: src/components/organisms/ChatSidebar/ChatSidebar.tsx [hunk 1]
```diff
@@ -57,6 +59,10 @@ export const _ChatSidebar: React.FC<ChatSidebarProps> = ({ venue }) => {
     toggleSidebar();
   }, [toggleSidebar]);
 
+  const goToAdmin = useCallback(() => {
```
Comment (goran-peoski-work) [inline src/components/organisms/ChatSidebar/ChatSidebar.tsx:62]: There are about 4 functions defined with name `goTo*` and 6 functions defined with `navigateTo*`. Could be useful if we settle on one of these conventions while it's easy to track  them.

File: src/components/organisms/ChatSidebar/ChatSidebar.tsx [hunk 2]
```diff
@@ -133,6 +139,17 @@ export const _ChatSidebar: React.FC<ChatSidebarProps> = ({ venue }) => {
...
 
+        {!isShownHelpCenter && !isExpanded && (
+          <button
+            aria-label={"Create space"}
+            className="chat-sidebar__controller chat-sidebar__create-icon"
+            onClick={goToAdmin}
+          >
+            <FontAwesomeIcon icon={faChevronLeft} size="sm" />
+            <FontAwesomeIcon icon={faPen} size="lg" />
+          </button>
```
Comment (goran-peoski-work) [inline src/components/organisms/ChatSidebar/ChatSidebar.tsx:150]: [OOS]

We should probably extract one component for all 4 of the existing sidebar buttons

File: src/components/organisms/ChatSidebar/ChatSidebar.tsx [hunk 3]
```diff
@@ -133,6 +139,17 @@ export const _ChatSidebar: React.FC<ChatSidebarProps> = ({ venue }) => {
           </button>
         )}
 
+        {!isShownHelpCenter && !isExpanded && (
+          <button
+            aria-label={"Create space"}
```
Comment (goran-peoski-work) [inline src/components/organisms/ChatSidebar/ChatSidebar.tsx:144]:
```suggestion
            aria-label="Create space"
```

**Review by goran-peoski-work — APPROVED** [review]
One minor adjustment suggested, and a note on naming. 

LGTM

File: src/components/organisms/ChatSidebar/ChatSidebar.tsx (hunk 2 above)
Comment (dmitrymarchuk) [reply src/components/organisms/ChatSidebar/ChatSidebar.tsx]: Doing that in https://github.com/sparkletown/sparkle/pull/2165

**Pull request merged**